### DISCUSSION TITLE
Prosto TV

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#185] [Show and tell]
Ukrainian host, this serves as a placeholder for future stream indications.
Service is free, gives access to hundreds of channels, and seems to be linked with a company named Ukrainian Telecommunication Group : https://utelecom.com.ua/

**Comment by Dum4G** [comment]
Very well known source that isn't used much for a reason. It bugs you with a paywall screen every 3 minutes, you have to wait for at least 30-40 seconds and reset it everytime
![image](https://user-images.githubusercontent.com/5430116/145563679-99d849f5-cd2b-42e4-bd4f-6d552efa62a3.png)
![image](https://user-images.githubusercontent.com/5430116/145563736-655a7642-d391-43db-aa46-3c7ca9588354.png)
https://github.com/xtinger/xtinger.github.io/blob/17e9e3eec08727bc1357a441eec436502aa9481f/IPTV/%D0%A3%D0%A2%D0%91060921.m3u

**Reply by BellezaEmporium** [reply]
I've seen that after a while indeed. Pretty weird way of doing it if I can say so myself x)
